src/stories/20_Features/Animations.html

```diff
@@ -258,57 +258,61 @@ <h1>zoom-out</h1>
 
     <!-- ## Pan-Left -->
     <amp-story-page id="pan-left">
-      <amp-story-grid-layer template="vertical">
-        <h1>pan-left</h1>
-        <amp-img animate-in="pan-left"
-                 animate-in-duration="4s"
-                 layout="fixed"
-                 src="https://picsum.photos/1024/768?image=1026"
-                 width="720"
-                 height="320">
-        </amp-img>
+      <amp-story-grid-layer template="fill">
+        <div>
```

Comment:
Why do we need the div here?

Reply:
Since the `amp-story-grid-layer` template=fill only applies for the first element, we must wrap all elements inside the page inside an element. I added a small text regarding this on the ken burns page, I'll add it here as well.